src/game-item.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { ChainID } from '@harmony-js/utils'
+import { expect, use } from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import sinon from 'sinon'
+import { GameItems } from './game-item'
+import { HarmonyShards } from './interfaces'
+import { PrivateKey } from './private-key'
+import { HRC1155_CONTRACT_ADDRESS, TEST_ACCOUNT_1 } from './tests/constants'
+import { ABI } from './tests/contracts/GameItems/abi'
+
+describe('Game Item Provider', () => {
+  use(chaiAsPromised)
+
+  let contract: GameItems
+  let provider: PrivateKey
+
+  before(() => {
+    provider = new PrivateKey(HarmonyShards.SHARD_0_TESTNET, TEST_ACCOUNT_1.privateKey, ChainID.HmyTestnet)
+    contract = new GameItems(HRC1155_CONTRACT_ADDRESS, ABI, provider)
+  })
+
+  afterEach(async () => {
+    sinon.restore()
+  })
+
+  it('should be defined', () => {
+    expect(contract).to.not.be.undefined
+  })
+
+  describe('getGold', () => {
+    it('should get the id of gold item', async () => {
+      const gold = await contract.getGold()
+      expect(gold).to.exist
+      expect(gold).to.not.be.null
+      expect(gold).to.not.undefined
+      expect(gold).to.be.equals(0)
+    })
+
+    it('should throw an error if something went wrong', async () => {
+      expect(contract.getGold()).to.be.rejectedWith(Error)
+    })
+  })
+
+  describe('getSilver', () => {
+    it('should get the id of silver item', async () => {
+      const silver = await contract.getSilver()
+      expect(silver).to.exist
+      expect(silver).to.not.be.null
+      expect(silver).to.not.undefined
+      expect(silver).to.be.equals(1)
```

Comment:
Use the token const from the [constants](https://github.com/blockcoders/harmony-marketplace-sdk/blob/main/src/tests/constants.ts)

Reply:
No problem!